File: official/docs/go/parcels/create_parcel.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	parcel, err := client.CreateParcel(
        &easypost.Parcel{
            Length: 20.2,
            Width:  10.9,
            Height: 5,
            Weight: 65.9,
        },
    )

	fmt.Println(parcel)
}

File: official/docs/java/parcels/CreateParcel.java
import com.easypost.EasyPost;
import java.util.*;

public class CreateParcel {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Map<String, Object> parcelMap = new HashMap<String, Object>();
        parcelMap.put("height", 5);
        parcelMap.put("width", 10.9);
        parcelMap.put("length", 20.2);
        parcelMap.put("weight", 65.9);

        Parcel parcel = Parcel.create(parcelMap);

        System.out.println(parcel);
    }
}
